parlai/mturk/core/dev/react_server/server/server.js

```diff
@@ -274,7 +274,7 @@ class LocalAgentState {
 // ======================= <Socket> =======================
 
 // Socket function types
-// TODO move to constants
+// WISH move to constants
```

Comment:
lol you being honest with yourself?

Reply:
i'm not sure WISH is in the common list of editor highlights/greps

https://github.com/JohnPostlethwait/fixme

Reply:
TIL the common codetags